containers/compose.elasticsearch.yaml

```diff
@@ -26,14 +31,14 @@ services:
       - bootstrap.memory_lock=true
       - "xpack.security.enabled=true"
       - "xpack.security.http.ssl.enabled=true"
-      - "xpack.security.http.ssl.key=/usr/share/elasticsearch/config/certs/privkey.pem"
-      - "xpack.security.http.ssl.certificate=/usr/share/elasticsearch/config/certs/fullchain.pem"
-      - "xpack.security.http.ssl.certificate_authorities=/usr/share/elasticsearch/config/certs/chain.pem"
+      - "xpack.security.http.ssl.key=/usr/share/elasticsearch/config/certs/privkey1.pem"
+      - "xpack.security.http.ssl.certificate=/usr/share/elasticsearch/config/certs/fullchain1.pem"
+      - "xpack.security.http.ssl.certificate_authorities=/usr/share/elasticsearch/config/certs/chain1.pem"
       - "xpack.security.http.ssl.verification_mode=certificate"
       - "xpack.security.transport.ssl.enabled=true"
-      - "xpack.security.transport.ssl.key=/usr/share/elasticsearch/config/certs/privkey.pem"
-      - "xpack.security.transport.ssl.certificate=/usr/share/elasticsearch/config/certs/fullchain.pem"
-      - "xpack.security.transport.ssl.certificate_authorities=/usr/share/elasticsearch/config/certs/chain.pem"
+      - "xpack.security.transport.ssl.key=/usr/share/elasticsearch/config/certs/privkey1.pem"
+      - "xpack.security.transport.ssl.certificate=/usr/share/elasticsearch/config/certs/fullchain1.pem"
+      - "xpack.security.transport.ssl.certificate_authorities=/usr/share/elasticsearch/config/certs/chain1.pem"
       - "xpack.security.transport.ssl.verification_mode=certificate"
```

Comment:
So, if you have the symbolic link bound to the updated version, there's no need to hardcode the current certificate version in the certificates directory::

where:
`/usr/share/elasticsearch/config/certs/chain1.pem`
will become:
`/usr/share/elasticsearch/config/certs/chain.pem`

Reply:
as I mentioned in the comment above, if I keep the link I would need to have another volume for the original files, otherwise the link will not work.

but I will change the strategy and I will copy the files, instead of the volume approach, so we can avoid the current issues with permissions.

thanks for taking time to review this PR